scripts/neutron-band-structure.py

```diff
@@ -0,0 +1,209 @@
+#! /usr/bin/env python3
+import argparse
+import os
+import pathlib
+from typing import List, Tuple
+
+import euphonic
+from euphonic import ureg
+import euphonic.plot
+import matplotlib.pyplot as plt
+import numpy as np
+import seekpath
+
+
+def get_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('file', type=str,
+                        help=('File with force constants data. Supported '
+                              'formats: .yaml (Phonopy); .castep_bin, .check '
+                              '(Castep); .json (Euphonic)'))
+    parser.add_argument('--ebins', type=int, default=200)
+    parser.add_argument('--seekpath-labels', action='store_true',
+                        dest='seekpath_labels',
+                        help='Use the exact labels reported by Seekpath when '
+                             'constructing the band structure path. Otherwise,'
+                             ' use the labels detected by Euphonic.')
+    parser.add_argument('--q-distance', type=float, default=0.025,
+                        dest='q_distance',
+                        help='Target distance between q-point samples')
+    parser.add_argument('--x-label', type=str, default=None,
+                        dest='x_label')
+    parser.add_argument('--y-label', type=str, default=None,
+                        dest='y_label')
+    parser.add_argument('--title', type=str, default=None)
+    parser.add_argument('--cmap', type=str, default='viridis',
+                        help='Matplotlib colormap')
+    parser.add_argument('--q-broadening', type=float, default=None,
+                        dest='gaussian_x',
+                        help='Width of Gaussian broadening on q axis in recip '
+                             'angstrom. (No broadening if unspecified.)')
+    parser.add_argument('--energy-broadening', type=float, default=None,
+                        dest='gaussian_y',
+                        help='Width of Gaussian broadening on energy axis in '
+                        'meV. (No broadening if unspecified.)')
+    return parser
+
+
+def get_seekpath_structure(crystal: euphonic.Crystal) -> Tuple[np.ndarray,
```

Comment:
This could be quite a useful function (maybe even as a method `Crystal.to_seekpath_structure`?) since there's already an example of doing this conversion in the docs and just calling this would be easier. Although it can probably be left here for now until we decide there's more uses for it

Reply:
I figured having it in the script is also an "example" of sorts. Maybe that's less true if we move scripts further into the module structure.